stdlib/config-api/src/test/java/org/ballerinalang/stdlib/config/ConfigTest.java

```diff
@@ -318,6 +318,28 @@ public void testBackwardSlashInString() throws IOException {
         Assert.assertEquals(returnVals[0].stringValue(), "C:\\Users\\John");
     }
 
+    @Test(description = "Test multiline config strings")
+    public void testMultilineConfigStrings() throws IOException {
+        registry.initRegistry(new HashMap<>(), null, ballerinaConfPath);
+        BValue[] inputArg = {new BString("multiline.strings.txt")};
+        BValue[] returnVals = BRunUtil.invoke(compileResult, "testGetAsString", inputArg);
+        Assert.assertEquals(returnVals[0].stringValue(),
+                            "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor " +
+                                    "incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis " +
+                                    "nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. " +
+                                    "Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu " +
+                                    "fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in " +
+                                    "culpa qui officia deserunt mollit anim id est laborum.");
+    }
+
+    @Test(description = "Test multiline config strings")
+    public void testMultilineConfigStrings2() throws IOException {
```

Comment:
Shall we add a meaningful name and a description? 
suggestion: testMultiSpacedConfigStrings